firebase-auth.html

```diff
@@ -45,6 +45,12 @@
 
 The `user` property will automatically be populated if an active session is
 available, so handling the resolved promise of sign-in methods is optional.
+
+It's important to note that if you're using a Service Worker you should let
+urls that contain `/__auth/` go through to the network, rather than have the
```

Comment:
It may also be worth noting that this is a particular problem when your Service Worker and your site are both served from a Firebase Hosting domain. If you are self-deploying your app to some non-Firebase domain, this shouldn't be a problem.